### ISSUE TITLE
user account management on Android

### ISSUE THREAD

**Issue by paradite** [kanghj/fyp-chrome-extension#3]
Hi, Hong Jin,

I noticed that you used the Chrome local storage for account management and [timestamp](https://github.com/kanghj/wordnews_chrome_extension/blob/master/contentscript.js#L623) for user id.

I am planning to use Android's internal ID as user ID and Android's app preference as storage. Would there be any issue on the backend with this approach?

**Comment by kanghj** [comment]
Yes, you should be able to use the internal ID as a user ID without any problems from the backend. Similarly, all of the preferences are actually handled on the client side. Actually, I was looking into implementing google login a few weeks ago. I'll try to work on it more next week. Will update you more when that happens.

**Comment by paradite** [comment]
Okay, will you be using https://developers.google.com/identity/ ?
Android has its own account management tools like https://developers.google.com/android/guides/overview

**Comment by kanghj** [comment]
Oops, sorry for late reply, didn't notice that you commented. Yes, I went with the first link (https://developers.google.com/identity/). The account identity should be usable on the Chrome extension too, so that an individual is treated as the same user on both the Chrome Extension and on the Android app, but I'm still working on that.

**Comment by paradite** [comment]
This Google Identity platform introduces a sign-in process. However, the currently proposed Android client should not have a sign-in process (the user should be able to use the app without signing in to Google account).
Hence, I will use some Android device ID first, and later provide an optional sign-in feature that links the device ID with Google Identity platform (via HTTP request and some backend logic) and sync with the accounts in your new chrome extension using Google Identity Platform.
